CodeSniffer.php

```diff
@@ -1933,6 +1933,10 @@ public static function isCamelCaps(
         $public=true,
         $strict=true
     ) {
+        if ('' === $string) {
```

Comment:
I'm not a Symfony developer. My intention of using a Yoda notations is to avoid `if ($string = '') { }`. I agree that Yoda notations are highly misused as in cases like `if (1 < $foo) { }` or `if ('' === trim($string))` it make no sense, still in this particular example Yoda notation can prevent a stupid typo which is quite hard to debug.

Saying all this I'm don't want to break your coding standards if they are strictly prohibiting Yoda notations. In this case please let me know and I will flip it.